### DISCUSSION TITLE
Scroll inside a pop up window

### DISCUSSION THREAD

**Discussion by abdurraheemj** [seleniumbase/SeleniumBase#1032] [Q&A] [answered]
Hi Michael,
I have problem scroll to bottom or scroll to element in pop up window. I would like to go with pure seleniumbase. Thank you so much for your answers

**Comment by mdmintz** [comment, marked as answer]
Hi @abdurraheemj  Similar to https://stackoverflow.com/a/45961672 , you could do:

```python
element_inside_popup = self.find_element(selector)
from selenium.webdriver.common.keys import Keys
element_inside_popup.send_keys(Keys.END)
```

**Reply by abdurraheemj** [reply]
self.find_element(css selector of my pop window)
getting an error saying not interactable

**Reply by mdmintz** [reply]
Use the selector of an element that’s inside the pop-up window.

**Reply by abdurraheemj** [reply]
Great! You are Genius! God Bless You for giving us this wonderful tool and for your support
